src/utils/evm.ts

```diff
@@ -0,0 +1,59 @@
+import { BigNumber, ethers } from 'ethers';
+import { verifySignedData } from '../signed-data-store';
+import { logger } from '../logger';
+import type { SignedData } from '../types';
+
+export const benchmark = async () => {
```

Comment:
This is not production code. Maybe make it a test? 

E.g. running 1K signatures and expecting it to finish in a reasonable amount of time? Not sure about this though since the timing largely depends on the machine capacity.